lib/src/rmui_bundle_suggestors/constants.dart

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// The script for the dev RMUI bundle.
+final rmuiBundleDev = ScriptToAdd(
+    path: 'packages/react_material_ui/react-material-ui-development.umd.js');
+
+/// The script for the prod RMUI bundle.
+final rmuiBundleProd =
+    ScriptToAdd(path: 'packages/react_material_ui/react-material-ui.umd.js');
+
+/// The script pattern for finding react-dart JS scripts.
+const reactJsScript = Script(pathSubpattern: r'packages/react/react\w*.js');
+
+/// A script that can be searched for via a script tag [pattern] for a
+/// specific path ([pathSubpattern]).
+class Script {
+  final String pathSubpattern;
+
+  const Script({required this.pathSubpattern});
+
+  /// A pattern for finding a script tag with a matching path,
+  /// including preceding whitespace and any path prefix.
+  ///
+  /// See:
+  ///
+  /// - [ScriptMatch.precedingWhitespaceGroup]
+  /// - [ScriptMatch.pathPrefixGroup]
+  RegExp get pattern => RegExp(
+      r'(?<preceding_whitespace>[^\S\r\n]*)<script.*src="(?<path_prefix>.*)' +
```

Comment:
I added the `path_prefix` part of the pattern and these classes with a `pathPrefix` arg to `scriptTag` so that we could preserve path prefixes in the matched React files, since during manual testing there were quite a few cases where it wasn't just `"packages/..."`.

Examples:
```html
<!-- No prefix -->
<script "packages/react/react.js"></script>

<!-- Absolute path -->
<script "/packages/react/react.js"></script>

<!-- Path to root of build in HTML file in subdirectory -->
<script "../packages/react/react.js"></script>

<!-- Templated URL -->
<script "{{BASE_URL}}packages/react/react.js"></script>
```